src/workers/communityLinkAccessMail.ts

```diff
@@ -0,0 +1,38 @@
+import { templateId } from '../common/mailing';
+import { messageToJson, Worker } from './worker';
+import { fetchUser } from '../common';
+import { baseNotificationEmailData, sendEmail } from '../common';
+
+interface Data {
+  userId: string;
+}
+
+// TODO:: Update the dynamic template data once available
```

Comment:
I guess this is done by now?

Reply:
The dangerous TODO comments 🤦
Will sort this out. Thank you!